services/core-web/src/tests/actionCreators/documentActionCreator.spec.js

```diff
@@ -29,20 +29,17 @@ describe("`documentActionCreator` action creator: exported Notice of Work ", ()
   };
 
   const url = `${ENVIRONMENT.apiUrl}${COMMON_API.NOW_APPLICATION_EXPORT_DOCUMENT_TYPE_OPTIONS}/${documentTypeCode}`;
-  it("Request successful, dispatches `success` with correct response", () => {
-    const mockResponse = { data: { token: "token" } };
-    mockAxios.onPost(url, payload).reply(200, mockResponse);
-    global.open = () => {
-      return {
-        onbeforeunload: () => jest.fn(),
-      };
-    };
+  it("Request failure, dispatches `error` with correct response", () => {
```

Comment:
I've spent roughly a few hours trying to fix this "successful" test for this action creator but I could not find a way for JEST to test methods of this shape (where the "then" of an axios request returns the promise another axios request).